Makefile

```diff
@@ -59,7 +59,9 @@ test: ## runs integration tests that require local applications such as MongoDB
 	go test -tags integration -race ./...
 
 bench: ## runs benchmark tests
-	go test -tags bench -benchmem -bench=. ./test/bench -memprofile=mem.prof -cpuprofile=cpu.prof | tee output.txt
```

Comment:
The exit status of benchmark test has been ignored, because the exit status of a pipeline is the exit status of the last command (`tee`) in the pipeline.
To reveal the original exit status, piping command is separated.